examples/grommet/src/FormWrapper.tsx

```diff
@@ -0,0 +1,302 @@
+          </>
+        )}
+      </Card>
+      <Card>
+        <CardBody background="white" pad="10px" width="1000px">
+          <div className="form-options">
+            <fieldset>
+              <Select
+                defaultValue="Editor"
+                options={['Editor', 'Viewer']}
+                value={isEditor ? 'Editor' : 'Viewer'}
+                onChange={(e) => {
+                  setIsEditor(e.target.value === 'Editor');
+                  setSchema({
+                    ...schema,
+                    uiSchema: {
+                      ...schema.uiSchema,
+                      'ui:submitButtonOptions': {
+                        norender: e.target.value === 'Editor',
+                      },
+                    },
+                  });
+                }}
+              />
+            </fieldset>
+            <fieldset className="form_language">
+              <Select
+                onChange={(e) => setLocale(e.target.value)}
+                options={[...VERNA_SUPPORTED_LOCALES]}
+                value={locale}
+              />
+            </fieldset>
+          </div>
+          <fieldset>
+            {pollSelected !== null ? (
+              <>
+                <Button
+                  className="form-goback-button"
+                  label={formatMessage(messages.goBack)}
+                  onClick={() => goToMainPage()}
+                />
+                <VernaForm onSubmit={onSubmit} />
+              </>
+            ) : (
+              <Poll
+                schemaForm={schemaForm}
+                setPollSelected={setPollSelected}
+                setSchemaForm={setSchemaForm}
+                stats={stats}
+              />
+            )}
+            {isEditor && (
+              <>
+                <Button
+                  label={formatMessage(messages.save)}
+                  margin={{ top: '15px' }}
+                  onClick={() => (pollSelected ? savePoll() : saveForm())}
+                />
+                <Button
+                  label={formatMessage(messages.resetData)}
+                  margin={{ left: '10px', top: '15px' }}
+                  onClick={() => {
+                    localStorage.clear();
+                    window.location.reload();
+                  }}
+                />
+              </>
+            )}
+          </fieldset>
+        </CardBody>
+        {formStatus === FormStatusEnum.completed && (
+          <CardFooter background={{ color: 'lightgreen' }} height="50px">
+            <Box
+              align="center"
+              direction="row"
+              gap="15px"
+              justify="start"
+              margin={{ left: '15px' }}
+            >
+              <StatusGood />
+              <Text>
+                <FormattedMessage {...messages.success} />
+              </Text>
+            </Box>
+          </CardFooter>
+        )}
+        {formStatus === FormStatusEnum.error && ( // TODO: activate it after refacto adding onError
```

Comment:
```suggestion
        {formStatus === FormStatusEnum.error && ( // TODO: activate it after refacto adding onError
```
this todo is not forgotten?

Reply:
Oops, it was for an old project of doing things, i removed it